packages/gatsby-remark-prismjs/src/load-prism-language-extension.js

```diff
@@ -0,0 +1,64 @@
+const Prism = require(`prismjs`)
+const loadPrismLanguage = require(`./load-prism-language`)
+const replaceStringWithRegex = require(`./replace-string-with-regexp`)
+
+module.exports = languageExtensions => {
+  //  We only accept array or object as the 'languageExtensions' config property
+  //  (Note that 'instanceof Object' will return true for both Array and Object)
+  if (!(languageExtensions instanceof Object)) {
```

Comment:
Why not just make it an array to begin with, then we can perform this check for each item?

e.g.

```
const extensions = [].concat(languageExtensions)
// rest of the code
```